docs/spire.md

```diff
@@ -0,0 +1,287 @@
+<!--
+---
+linkTitle: "TaskRun Result Attestation"
+weight: 1660
```

Comment:
We're in the process of restructuring the documentation, so we don't have a good framework for assigning these IDs right now. I imaging 1660 might end up towards the end of the list.

Reply:
This is good point. But I am not sure how to address this?  Should we lower the weight to say somewhere below 1000s?